client/src/featureform/register.py

```diff
@@ -2999,6 +3102,7 @@ def list_providers(self, local=False):
 clear_state = global_registrar.clear_state
 register_user = global_registrar.register_user
 register_redis = global_registrar.register_redis
+register_blob_store = global_registrar.register_blob_store
```

Comment:
do we need to add `get_azure_blob` here as well? And should both functions be named similarly?